simple_switch/control_plane/controllers/int/report/report.proto

```diff
@@ -0,0 +1,134 @@
+// Copied from github.com/lschulz/p4-examples/telemetry/kafka/report
```

Comment:
Replace this file with a relative symlink (ln -sr) to telemetry/kafka/report/report.proto

Reply:
Done.